source/enum_set.h

```diff
@@ -149,25 +271,31 @@ class EnumSet {
     return bucket.data & ComputeMaskForValue(value);
   }
 
+  // Wrapper on `Contains` which matches the STL API for sets.
```

Comment:
The comment should describe what the function does, not how it is implemented.

```suggestion
  // Returns the number of times `value` is in the set.
```